src/serializer.js

```diff
@@ -726,8 +765,9 @@ var Serializer = (function () {
       var results = []
       var type, t, st, pred
       var sts = stats.subjects[this.toStr(subject)] // relevant statements
-      if (typeof sts === 'undefined') {
-        throw new Error('Serializing XML - Cant find statements for ' + subject)
+      if (typeof sts === 'undefined') { // empty bnode
+        return propertyXMLTree(subject, stats)
```

Comment:
Yes, an addition to structure.* of a a bnode with a sole incoming arc   like  { :MLK :friend [].  }.  That used not to serialize in XML without throwing the errors.  Added  also another combo:  { [] :loves []. } as a related test.